ajax/libs/Modaal/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Modaal",
+  "description": "An accessible dialog window library for all humans.",
+  "version": "0.3.1",
+  "filename": "modaal.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/humaan/Modaal"
+  },
+  "license": "MIT",
+  "keywords": [
+    "modaal"
+  ],
+  "npmName": "modaal",
+  "npmFileMap": [
+    {
```

Comment:
Let's use git auto-update config
Because I found that there are 2 releases versions from npm, and 7 releases versions from GitHub.
As usual, we prefer to use more releases versions as the auto-update option.

Let's modify from A to B.
- A
```js
  "npmName": "modaal",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*"
      ]
    }
  ]
```
- B
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/humaan/Modaal.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/!(*.scss)"
        ]
      }
    ]
  }
```

Thanks